mtags/src/main/scala-3/scala/meta/internal/pc/AutoImports.scala

```diff
@@ -33,6 +34,12 @@ object AutoImports:
      */
     case Renamed(sym: Symbol, ownerRename: String)
 
+    /**
+     * Rename symbol itself, import only.
+     * `Map -> ("", import java.{util => ju})`
+     */
+    case SelfRenamed(sym: Symbol, name: String)
```

Comment:
Why do we need a separate case to `Renamed`?

Reply:
Because `Renamed(<java.util.Optional>, "ju")` is for when we complete `ju.Optional` auto importing `java.{util => ju}`. On the other hand, `SelfRenamed(<java.util>, "ju")` represents, import `java.{util => ju}` without completing anything.

`Renamed` and `SelfRenamed` expect to have different symbols, maybe we should add a comment explaining this.